src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -47,56 +55,73 @@ export class CheckBox extends React.Component<Props> {
   };
 
   private getComponentStyle = (style: StyleType): StyleType => {
-    const { select, highlight, ...container } = style;
+    const { text, select, highlight, ...container } = style;
 
     return {
-      container: {
-        ...container,
-        ...styles.container,
-      },
-      select: {
-        width: container.width / 2,
-        height: container.height / 8,
-        ...select,
-        ...styles.select,
-      },
-      highlight: {
-        ...highlight,
-        ...styles.highlight,
-      },
+      selectContainer: container,
+      select: select,
+      highlight: highlight,
+      text: text,
     };
   };
 
+  private renderTextElement = (style: StyleType): React.ReactElement<TextProps> => {
+    const { text } = this.props;
+
+    return (
+      <Text style={[style, styles.text]} key={0}>{text}</Text>
+    );
+  };
+
+  private renderComponentChildren = (style: StyleType): React.ReactNode => {
+    const { text } = this.props;
+
+    return [
```

Comment:
Can it be just Text nut not an array for this aims?

Reply:
Yes, but you know about historically builded code style 😄